### ISSUE TITLE
Message: Interface\AddOns\Plater\Plater.lua:7614: attempt to index local 'platerRules' (a boolean value)

### ISSUE THREAD

**Issue by tomsommer** [Tercioo/Plater-Nameplates#83]
```
Message: Interface\AddOns\Plater\Plater.lua:7614: attempt to index local 'platerRules' (a boolean value)
Time: Sun Jan 24 16:06:47 2021
Count: 1
Stack: Interface\AddOns\Plater\Plater.lua:7614: attempt to index local 'platerRules' (a boolean value)
[string "@Interface\AddOns\Plater\Plater.lua"]:7614: in function <Interface\AddOns\Plater\Plater.lua:7580>

Locals: fromInit = nil
platerThemeName = "Plater Nameplates Theme"
platerRuleName = "Plater Nameplates Rule"
platerTheme = <table> {
 textStyles = <table> {
 }
 fontFace = "Fonts\FRIZQT__.TTF"
 timerOffset = 0
 drawSwipes = true
 scaleText = true
 tenthsDuration = 0
 fontOutline = "OUTLINE"
 minSize = 0.500000
 mmSSDuration = 0
 enableText = false
 effect = "pulse"
 yOff = 0
 xOff = 0
 fontSize = 18
 fontShadow = <table> {
 }
 minEffectDuration = 30
 minDuration = 2
 anchor = "CENTER"
}
platerRules = false
(*temporary) = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:287
(*temporary) = <table> {
 4 = <table> {
 }
}
(*temporary) = 0
(*temporary) = nil
(*temporary) = nil
(*temporary) = 0
(*temporary) = "attempt to index local 'platerRules' (a boolean value)"
DF = <table> {
 CooldownToClass = <table> {
 }
 ClassCache = <table> {
 }
 SavePositionOnScreen = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:2139
 CreateTabContainer = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:3907
 IsUnitTapDenied = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:4028
 CancelTimer = <function> defined @Interface\AddOns\ArkInventory\Externals\Ace3\AceTimer\AceTimer-3.0.lua:147
 NewLabel = <function> defined @Interface\AddOns\Details\Libs\DF\label.lua:275
 SavedVars = <table> {
 }
 BuildStatusbarAuthorInfo = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:6972
 PanelCounter = 583208
 IconPick = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:1515
 RuneIDs = <table> {
 }
 LabelNameCounter = 583371
 ListboxFunctions = <table> {
 }
 GetCurrentSpec = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:3325
 alias_text_colors = <table> {
 }
 HidaAllBarrasInfo = <function> defined @Interface\AddOns\Details\frames\window_playerbreakdown.lua:421
 UnitFrameFunctions = <table> {
 }
 ScheduleTimer = <function> defined @Interface\AddOns\ArkInventory\Externals\Ace3\AceTimer\AceTimer-3.0.lua:94
 CreateCastBar = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:8575
 RegisterScriptComm = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:3970
 ShowPanicWarning = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:1908
 UnitGroupRolesAssigned = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:59
 SetAutoCompleteWithSpells = <function> defined @Interface\AddOns\Details\Libs\DF\auras.lua:1061
 ShowTutorialAlertFrame = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:1907
 CooldownsExternals = <table> {
 }
 GetSpecializationRole = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:95
 SpecListByClass = <table> {
 }
 FRAMELEVEL_BACKGROUND = 150
 CreateHealthBar = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:7434
 RunHooksForWidget = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:2296
 GetDefaultColorList = <function> defined @Interface\AddOns\Details\Libs\DF\colors.lua:170
 button_templates = <table> {
 }
 CreateNewDropdownFrame = <function> defined @Interface\AddOns\Details\Libs\DF\dropdown.lua:1137
 ReskinSlider = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:3184
 CriaNovaBarraInfo2 = <function> defined @Interface\AddOns\Details\frames\window_playerbreakdown.lua:5775
 ColorPick = <function> defined @Interface\AddOns\Details\Libs\DF\panel.lua:1491
 CreateSwitch = <function> defined @Interface\AddOns\Details\Libs\DF\slider.lua:966
 font_templates = <table> {
 }
 embeds = <table> {
 }
 SetAnchor = <function> defined @Interface\AddOns\Details\Libs\DF\fw.lua:850
 trim = <function> defined @Interface\AddOns\Details\Libs\DF\
```

**Comment by cont1nuity** [comment]
Please check if your installed OmniCC is up to date.

**Comment by tomsommer** [comment]
OmniCC 9.0.5

**Comment by cont1nuity** [comment]
Please try removing existing Plater rules from OmniCC and do a /reload.

**Comment by tomsommer** [comment]
Doesn't work, same error

**Comment by cont1nuity** [comment]
Please export your OmniCC profile and link/attach it here.

**Comment by tomsommer** [comment]
Toggled the Plater Aura Timer setting and OmniCC setting, then an OmniCC rule appeared and I deleted that. Solved it.

**Comment by cont1nuity** [comment]
Thank you. I will still investigate this and see if the issue can be mitigated.

**Comment by mbarberot** [comment]
Hello, had this issue with OmniCC 9.0.8 and Plater 9.0.2.326b
As expected from the previous comments, I did not had the Plater Nameplates Rule in OmniCC rules.
Toggling the plater aura timer settings did not work.

To get the rule in OmniCC, I had to 
1. uninstalling Plater
2. creating a blank OmniCC profile
3. reinstalling Plater

I wonder if this could be related with having an "old" OmniCC profile where Plater could not add the rule for some reason 🤔
